src/pages/index.astro

```diff
@@ -1,33 +1,125 @@
 ---
 import MainLayout from '../layouts/MainLayout.astro';
+import {COMMUNITY_INVITE_URL, GITHUB_ORG_URL, REDDIT_URL} from '../consts';
 ---
 
 <MainLayout title="" }>
-	<div class="logo">
-		<img src="/syscraft-logo.gif" alt="Syscraft logo"/>
-		Syscraft
-	</div>
-</MainLayout>
+    <div class="logo">
+        <img src="/syscraft-logo.gif" alt="Syscraft logo"/>
+        Syscraft
+    </div>
+
+    <h1>Welcome</h1>
```

Comment:
I think maybe this content should be in MD format? Not really sure how to go about that though.

Reply:
oh nvm, I saw your comment about this in the PR description